### PR TITLE
feat(server): integration controllers

### PR DESCRIPTION
**Pull request by yk-eukarya**
# Overview

### CODE REVIEW

File: server/pkg/integration/integration.go [hunk 1]
```diff
@@ -71,7 +71,7 @@ func (i *Integration) SetDeveloper(developer UserID) {
 	i.developer = developer
 }
 
-func (i *Integration) Webhook() []*Webhook {
+func (i *Integration) Webhooks() []*Webhook {
 	return i.webhook
```
Comment (rot1024) [inline server/pkg/integration/integration.go:75]:
```suggestion
	return slices.Clone(i.webhooks)
```

rename `webhook` filed to `webhooks`

File: server/internal/adapter/gql/resolver_mutation_integration.go [hunk 2]
```diff
@@ -4,28 +4,150 @@ import (
...
+	if err != nil {
+		return nil, err
+	}
+
+	return &gqlmodel.DeleteWebhookPayload{
+		WebhookID: input.WebhookID,
+	}, nil
+}
+
+func FromPtr[T any](pointer *T, defaultValue T) T {
```
Comment (rot1024) [inline server/internal/adapter/gql/resolver_mutation_integration.go:148]: should use lo.FromPtrOr https://github.com/samber/lo#fromptror

File: server/internal/adapter/gql/gqlmodel/convert_integration.go [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+		Name:        i.Name(),
+		Description: lo.ToPtr(i.Description()),
+		LogoURL:     *i.LogoUrl(),
+		IType:       ToIntegrationType(i.Type()),
+		DeveloperID: IDFrom(i.Developer()),
+		Developer:   nil,
+		Config: &IntegrationConfig{
+			Token:    i.Token(),
+			Webhooks: ToWebhooks(i.Webhooks()),
+		},
```
Comment (rot1024) [inline server/internal/adapter/gql/gqlmodel/convert_integration.go:25]: If it is not the user's integration, config should be nil.

**Review by rot1024 — CHANGES_REQUESTED** [review]

**Comment by yk-eukarya** [comment]
Closed by #240

**Pull request closed without merging**